shell/src/main/java/alluxio/shell/command/TailCommand.java

```diff
@@ -56,15 +60,24 @@ void runCommand(AlluxioURI path, CommandLine cl) throws IOException {
       throw new IOException(e.getMessage());
     }
 
+    int numOfBytes = -1;
```

Comment:
We could set this to `Constants.KB` and not have to keep track of `userSpecifiedBytes`